### PR TITLE
LPS-153813 LPS-167012  Move test logic to generated classes

### PR DESCRIPTION
**Pull request by wcao20170619**

### CODE REVIEW

**Comment by wcao20170619** [comment]
ci:test:search

**Comment by wcao20170619** [comment]
ci:test:sf

File: modules/dxp/apps/search-experiences/search-experiences-rest-test/src/testIntegration/java/com/liferay/search/experiences/rest/resource/v1_0/test/SearchIndexResourceTest.java [hunk 1]
```diff
@@ -15,14 +15,76 @@
...
-@Ignore
 @RunWith(Arquillian.class)
 public class SearchIndexResourceTest extends BaseSearchIndexResourceTestCase {
+
+	@Override
+	@Test
+	public void testGetSearchIndexesPage() throws Exception {
+		Page<SearchIndex> page = searchIndexResource.getSearchIndexesPage();
+
+		List<SearchIndex> searchIndexs = _getSearchIndexs();
```
Comment (peerkar) [inline modules/dxp/apps/search-experiences/search-experiences-rest-test/src/testIntegration/java/com/liferay/search/experiences/rest/resource/v1_0/test/SearchIndexResourceTest.java:45]: @wcao20170619 typo searchIndexs

**Comment by wcao20170619** [comment]
ci:test:sf

**Comment by wcao20170619** [comment]
ci:test:search

**Comment by wcao20170619** [comment]
ci:test:sf

**Comment by wcao20170619** [comment]
ci:test:search

**Comment by wcao20170619** [comment]
ci:test:relevant

**Comment by wcao20170619** [comment]
ci:test:relevant

**Comment by wcao20170619** [comment]
ci:forward

**Pull request closed without merging**